pyproject.toml

```
@@ -14,18 +14,18 @@ classifiers = [
 dependencies = [
     'asdf >=2.15.0',
     'asdf-astropy >=0.4.0',
-    'astropy >=5.0.4',
+    'astropy >=5.3.0',
     'crds >=11.16.16',
     'gwcs >=0.18.1',
     'jsonschema >=4.8',
     'numpy >=1.22',
     'photutils >=1.6.0',
     'pyparsing >=2.4.7',
     'requests >=2.22',
-    'rad == 0.17.0',
+    'rad == 0.17.1',
```

```suggestion
    'rad >=0.17.1',
```
This should not be exact pinned.